packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -57,6 +57,13 @@ export interface HttpRouteMatch {
    * @default - do not match on HTTP2 request protocol
    */
   readonly protocol?: HttpRouteProtocol;
+
+  /**
+   * The query parameter to match on.
```

Comment:
Can you explain what multiples mean here? Are they `or`ed, or `and`ed?

Reply:
Specified that this is `and`.